### DISCUSSION TITLE
Will this work on Uno Platform?

### DISCUSSION THREAD

**Discussion by StoutOne** [MahApps/MahApps.Metro#4216] [General]
Hi. Can this work on Uno Platform, specifically the UWP and Linux platforms? Or is it only for the WPF platform?

Thanks.

**Comment by timunie** [comment]
Hi @StoutOne

Unfortunately MahApps is WPF only. For Linux I would have a look at Avalonia, which is multiplatform out of the box, but they have a different approach than Uno. Uno uses platform controls where as Avalonia uses its own with custom Styling support.

Happy coding
Tim

**Reply by StoutOne** [reply]
Hi. Good advice; I'll check it out. Thank you.
